examples/arm/setup.sh

```diff
@@ -215,7 +215,7 @@ function setup_vela() {
     if [[ ! -e ethos-u-vela ]]; then
         git clone https://review.mlplatform.org/ml/ethos-u/ethos-u-vela
         repo_dir="${root_dir}/ethos-u-vela"
-        base_rev=7706c1281166e7611f4300ed26338087152a33c9
+        base_rev=d362f5443f67b1e6213a9d8f124edff758efac96
```

Comment:
seems like we are regressing a bit on U55, is that a concern when we move to the new backend?

Reply:
No, not really.

Reply:
How do you mean regressing Digant?
and Per, do you mean no we're not regressing or no we're not concerned? :)

Reply:
Understood, thanks.